dev-db/pgcli/pgcli-3.1.0.ebuild

```diff
@@ -31,7 +31,6 @@ BDEPEND="
 	test? (
 		dev-db/postgresql
 		dev-python/mock[${PYTHON_USEDEP}]
-		dev-python/python-dateutil[${PYTHON_USEDEP}]
```

Comment:
If it directly needs dateutil, keep it in

Reply:
Actually `pendulum` needs it, but didn't pull it in due to DEPEND issue